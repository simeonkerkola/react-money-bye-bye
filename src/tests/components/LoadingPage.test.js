import React from 'react'
import { shallow } from 'enzyme'
import LoadingPage from '../../components/LoadingPage'

test('expect Loading Page to render correcly', () => {
  const wrapper = shallow(<LoadingPage />)
  expect(wrapper).toMatchSnapshot()
})
