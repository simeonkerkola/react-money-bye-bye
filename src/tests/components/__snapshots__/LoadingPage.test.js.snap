// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect Loading Page to render correcly 1`] = `
<div
  className="loader"
>
  <img
    alt="Loading..."
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
